caproto/tests/test_examples.py

```diff
@@ -343,6 +343,7 @@ def find_put_value(pv):
     [('caproto.ioc_examples.all_in_one', 'MyPVGroup',
       dict(macros={'macro': 'expanded'})),
      ('caproto.ioc_examples.chirp', 'Chirp', {'ramprate': 0.75}),
+     ('caproto.ioc_examples.thermo_sim', 'Thermo', {'period': 0.75}),
```

Comment:
`period` kwarg here is causing test failures

Reply:
you beat me to fixing this by about 4 minutes